modules/core/src/main/scala/lucuma/odb/api/repo/ObservationRepo.scala

```diff
@@ -265,29 +233,41 @@ object ObservationRepo {
               edit.observationIds,
               State.pure[ObservationModel, Unit](()),
               e.fold(Input.unassign[Asterism.Id])(_.fold(aid => Input(aid), _ => Input.ignore[Asterism.Id])),
-              e.fold(Input.unassign[Target.Id])(_.fold(_ => Input.ignore[Target.Id], tid => Input(tid))),
-              Input.ignore[ConstraintSet.Id]
+              e.fold(Input.unassign[Target.Id])(_.fold(_ => Input.ignore[Target.Id], tid => Input(tid)))
             )
         } yield os
 
-      override def editConstraintSet(
-        edit: ObservationModel.EditConstraintSet
-      ): F[List[ObservationModel]] =
-          doEdit(
-            edit.observationIds,
-            State.pure[ObservationModel, Unit](()),
-            Input.ignore[Asterism.Id],
-            Input.ignore[Target.Id],
-            edit.constraintSetId.fold(Input.unassign[ConstraintSet.Id])(Input(_))
-          )
-
       override def edit(
         edit: ObservationModel.Edit
       ): F[ObservationModel] =
 
         (edit.editor, edit.pointing).mapN { case (e, (a, t)) =>
-          doEdit(List(edit.observationId), e, a, t, edit.constraintSetId)
+          doEdit(List(edit.observationId), e, a, t)
         }.liftTo[F].flatten.map(_.head)
 
+
+      override def bulkEditConstraintSet(
+        edit: ConstraintSetModel.BulkEdit
```

Comment:
One nice thing, I think, is that you can bulk edit the constraint sets of observations whether or not they share constraint sets in common.  For example, you could set the sky background to `GRAY` across all observations regardless of what constraints they currently hold.

Reply:
Cool, I guess that leaves the other constraints as they are for those observations

Reply:
Now we need to support multi-select!

Reply:
Is it ever the case that a human API-user will want to just update all observations in the program?  I could add an alternative to specify the program id instead of a list of observation ids if so.